templates/node_inspector_attributes.html

```diff
@@ -1,112 +1,25 @@
 <div class="card">
```

Comment:
I think this file is not included (in node_inspector.html) anymore. So I think we can remove the whole file.